jina/optimizers/__init__.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from ruamel.yaml import YAML
+
+from .new_flow_runner import MultiFlowRunner
+from jina.helper import colored
+from jina.logging import default_logger as logger
+from .parameters import load_optimization_parameters
+
+
+class EvaluationCallback:
```

Comment:
I know, it is just a draft but before I forget to mention it - We need a documentation on class level and public method level.

Reply:
Yes, the docstring is a must-have.